modules/util/loss/masked_loss.py

```diff
@@ -2,12 +2,7 @@
 from torch import Tensor
 
 
-def masked_losses(
-        losses: Tensor,
-        mask: Tensor,
-        unmasked_weight: float,
-        normalize_masked_area_loss: bool
-) -> Tensor:
+def masked_losses(losses: Tensor, mask: Tensor, unmasked_weight: float, normalize_masked_area_loss: bool) -> Tensor:
```

Comment:
Another case of manually inserted line breaks that were removed. This shouldn't happen.